tools/js-tools/git-hooks/pre-push-hook.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+
+/* eslint-disable no-console, no-process-exit */
+const { spawnSync } = require( 'child_process' );
+const chalk = require( 'chalk' );
+
+console.log( chalk.green( 'Checking if changelog files are needed. Just a sec...' ) );
+
+/**
+ * Checks if changelog files are required.
+ */
+function checkChangelogFiles() {
+	const needChangelog = spawnSync(
+		'tools/check-changelogger-use.php',
+		[ 'origin/master', 'HEAD' ],
+		{
+			stdio: 'inherit',
```

Comment:
Just for good measure, let's make sure the cwd is as expected.
```suggestion
			stdio: 'inherit',
			cwd: __dirname + '/../../../',
```